WordPress/WordPressShareExtension/NSExtensionContext+Extensions.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+/// Encapsulates NSExtensionContext Helper Methods.
+///
+extension NSExtensionContext
+{
```

Comment:
Updated! i could have sworn that for classes / structs, the brace goes always on a new line.
Thanks for the heads up!